__tests__/app.test.js

```diff
@@ -83,3 +83,27 @@ describe('GET - /api/reviews/1000000', () => {
         })
     })
 })
+
+describe('GET - /api/reviews', () => {
+    test('/api/reviews - status 200 - responds with an array', () => {
+        return request(app).get('/api/reviews').expect(200).then(result => {
+            expect(Array.isArray(result.body.reviews)).toBe(true)
+        })
+    })
+    test('/api/reviews - status 200 - responds with an array of object, each object should have 9 keys', () => {
+        return request(app).get('/api/reviews').expect(200).then(result => {
+            result.body.reviews.forEach(review => {
+                expect(Object.keys(review).length).toBe(9)
+                expect(typeof review.owner).toBe('string')
+                expect(typeof review.title).toBe('string')
+                expect(typeof review.review_id).toBe('number')
+                expect(typeof review.category).toBe('string')
+                expect(typeof review.review_img_url).toBe('string')
+                expect(typeof review.created_at).toBe('string')
+                expect(typeof review.votes).toBe('number')
+                expect(typeof review.designer).toBe('string')
+                expect(typeof review.comment_count).toBe('string')
+            })
+        })
+    })
```

Comment:
There is no test to show that this has been sorted into date order.

Reply:
I put this on my pr request, jest-sorted wouldn't work with the utils?